Loop/Models/LoopConstants.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LoopConstants.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 10/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKit
+
+struct LoopConstants {
```

Comment:
👍 Thanks for doing this.

[nit] if this is only purely ever a bunch of static values, with namespace `LoopConstants`, consider changing this to:

```swift
enum LoopConstants
```

to ensure `LoopConstants` can't be `init()`ed.  

(You don't have to change anything else...the `static` values here are fine)